psydac/api/tests/test_assembly.py

```diff
@@ -532,6 +531,8 @@ def test_assembly_no_synchr_args(backend):
 
 #==============================================================================
 if __name__ == '__main__':
+    test_Norm_complex(None)
+    exit()
```

Comment:
With this `exit()` the rest of the tests are not run.

Reply:
this is only executed when the file is run directly (for debugging I presume), not in the tests.